Clicker/Models/Parsers/PollParser.swift

```diff
@@ -17,6 +17,10 @@ class PollParser: Parser {
         let id = json[ParserKeys.idKey].intValue
         let text = json[ParserKeys.textKey].stringValue
         let results = json[ParserKeys.resultsKey].dictionaryValue
+        results.forEach { (key, value) in
```

Comment:
is this still needed or can it be removed now?

Reply:
Not needed anymore, so I'll remove that. Thanks!